src/tests/unit_asio_stream.cpp

```diff
@@ -0,0 +1,560 @@
+#include "tests.h"
```

Comment:
There should be a copyright/license declaration at the top if this file as well.